src/main/java/io/reflect/token/Parameter.java

```diff
@@ -1,11 +1,14 @@
 package io.reflect.token;
 
-import com.google.gson.Gson;
-import com.google.gson.GsonBuilder;
-
+import java.io.IOException;
 import java.util.Arrays;
 
-public class Parameter {
+import net.minidev.json.JSONAwareEx;
+import net.minidev.json.JSONObject;
+import net.minidev.json.JSONStreamAwareEx;
+import net.minidev.json.JSONStyle;
```

Comment:
Just curious--why move away from `gson`?

Reply:
This is what the JWT library uses internally, so I didn't really have a choice.